frontend/packages/dev-console/integration-tests/features/virtulization/actions-on-Vm.feature

```diff
@@ -0,0 +1,84 @@
+Feature: Perform Actions on created VM
+    User should be able to perform Actions on imported VM
+
+
+Background: 
+    Given user is at Developer perspective
+    And user has already created VM
+
+
+@regression
+Scenario: Sidebar for VM: VM-03
+    Given user is at Developer Perspective
+    And user is having VM on the Topology page
+    When user clicks on the VM to open the sidebar
+    Then user can see the Details tab
+    And user can see the Resources tab
+    And user can see the Actions dropdown
+
+
+@regression
+Scenario: Edit Application Groupings action on VM: VM-04-TC02
+    Given user is at Developer Perspective
+    And user is having VM on the Topology page
+    When user right clicks on the VM to open the context menu
+    And user clicks on the Edit Application Groupings
+    Then user will see a modal
+    And user will see a Application dropdown
+
+
+@regression
+Scenario: Start VM action on VM: VM-04-TC03
+    Given user is at Developer Perspective
+    And user is having VM on the Topology page
+    When user right clicks on the VM to open the context menu
+    And user clicks on the Start VM
+    Then user will see Stop VM item in context menu
```

Comment:
Can't we make it to one step instead of 3 steps

Reply:
We can but then I need add those option comma separated. I will do that.